SpatialGDK/Source/SpatialGDKEditor/Private/SnapshotGenerator/SpatialGDKEditorSnapshotGenerator.cpp

```diff
@@ -59,7 +59,8 @@ bool CreateSpawnerEntity(Worker_SnapshotOutputStream* OutputStream)
 	SpawnerEntity.component_count = Components.Num();
 	SpawnerEntity.components = Components.GetData();
 
-	return Worker_SnapshotOutputStream_WriteEntity(OutputStream, &SpawnerEntity) != 0;
+	Worker_SnapshotOutputStream_WriteEntity(OutputStream, &SpawnerEntity);
+	return Worker_SnapshotOutputStream_GetState(OutputStream).stream_state == WORKER_STREAM_STATE_GOOD;
```

Comment:
We might want to log on this failing.

Reply:
This has a corresponding log in SpatialGDKEditorSnapshotGenerator::FillSnapshot, but maybe could be refactored.